examples/tutorials/workgraphs/scales_workgraph.ipynb

```diff
@@ -0,0 +1,469 @@
+    "        \"train_file\": files[\"train_file\"],\n",
+    "        \"valid_file\": files[\"valid_file\"]\n",
+    "    }\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d3396d54",
+   "metadata": {},
+   "source": [
+    "Setup the QE inputs these variables can be changed to your configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a35e9ed6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qe_inputs = {\n",
+    "    \"task_metadata\": Dict({\n",
+    "            \"options\": {\n",
+    "                \"resources\": {\n",
+    "                    \"tot_num_mpiprocs\":1,\n",
+    "                    'num_mpiprocs_per_machine':1,\n",
+    "                    'num_cores_per_mpiproc':8,\n",
+    "                },\n",
+    "                \"max_wallclock_seconds\": 3600,\n",
+    "                \"queue_name\": \"scarf\",\n",
+    "                \"qos\": \"scarf\",\n",
+    "                \"environment_variables\": {},\n",
+    "                \"withmpi\": True,\n",
+    "                \"prepend_text\": \"\"\"\n",
+    "                \"\"\",\n",
+    "                \"append_text\": \"\",\n",
+    "            },\n",
+    "    }),\n",
+    "    \"kpoints_mesh\": List([1, 1, 1]),\n",
+    "    \"code\": qe_code,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9074302a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ase.build import bulk\n",
+    "\n",
+    "with WorkGraph(\"EOS_workflow\") as wg:\n",
+    "\n",
+    "    initial_structure = StructureData(ase=bulk(\"NaCl\", \"rocksalt\", 5.63))\n",
+    "\n",
+    "    scales_task = wg.add_task(\n",
+    "        create_scales,\n",
+    "        min_v=0.95,\n",
+    "        max_v=1.05,\n",
+    "        num_structs=12,\n",
+    "        structure=initial_structure\n",
+    "    )\n",
+    "\n",
+    "    qe_task = wg.add_task(\n",
+    "        qe,\n",
+    "        **qe_inputs,\n",
+    "        scaled_structures=scales_task.outputs.scaled_structures\n",
+    "    )\n",
+    "    \n",
+    "    train_task = wg.add_task(\n",
+    "        create_train_files,\n",
+    "        structures=qe_task.outputs.structures\n",
+    "    )\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f97d4126",
+   "metadata": {},
+   "source": [
+    "Visualise and run `WorkGraph`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "59425796",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f230babb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f8a6b9ee",
+   "metadata": {},
+   "source": [
+    "We can can use the outputs to viusualise the data, in this example we will print a simple bar chart of the energies:"
```

Comment:
It would also be nice to actually have the EoS plot here (see examples linked elsewhere). All you need is the energies (from the DFT) and the volumes (which you can get from the structures if this is not saved).

There's probably no time right now, but it would also be cool to then do an `eos` via janus with the fine-tuned model, and see how that compares.